corehq/apps/accounting/tests/generator.py

```diff
@@ -39,9 +38,11 @@
     SmsUsageFeeCriteria,
 )
 from corehq.apps.users.models import WebUser, CommCareUser
+from corehq.util.test_utils import unit_testing_only
```

Comment:
> Test code should always reside inside a test package and should never be imported by production code

I agree with this, though I still think `unit_testing_only` is still a useful thing to have. The common way I've used it is to prevent something like `delete_all_cases` from being run outside of a test, which I think is a good protection to have, even if all the test code is in its own package. If someone accidentally ran some of those functions on a production machine that would be devastating.